ISSUES.md

```diff
@@ -0,0 +1,2 @@
+###Jira issues
+Jira functionality is not active now. In cause of changes in Jira EPAM authentication prevents connections from API.
```

Comment:
Please provide a little detail. You can use the ticket https://support.epam.com/esp/ess.do?ctx=docEngine&file=incidents&query=incident.id%3D%22SD4449964%22&title=Interaction%20SD4449964

Reply:
Added more details